CI/beta-release-script.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -x
+
+cat <<EOF > /tmp/version.py
+
+import json
+import sys
+
+with open("./package.json", "r") as package:
+    package_info = json.loads(package.read())
+    version = package_info["version"]
+
+    print(version)
+
+EOF
+
+VERSION=$(python3 /tmp/version.py)
+sed -r -i "s/\"version\": \"[^\"]+\"/\"version\": \"$VERSION\"/g" bower.json
+sed -r -i "s/\"version\": \"[^\"]+\"/\"version\": \"$VERSION\"/g" package.json
```

Comment:
you can put both filenames on the same line as sed is using all remaining arguments as source files.